scripts/repeatIT.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+#
+# Copyright Kroxylicious Authors.
+#
+# Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+#
+
+targetTest=${1:-ExpositionIT}
+
+logFile=/tmp/mvn-${targetTest}-$(date +"%d-%m-%Y_%H:%M").log
```

Comment:
Something that occurred to me just now is that the special characters (`-`, `:`) in the date makes it (slightly) harder to retrieve the file via command line.
The way I usually get around this while keeping things readable is to use ISO 8601 format but drop the punctuation, i.e. `$(date +"%Y%m%dT%H%M%S")` for ISO timestamp `2023-07-26T16:15:30` produces `20230726T161530` which is still mostly parseable by eye.

Reply:
updated in 3d7d5b3ba4bb8d5fd49ffb4e7cc46e2a4a917a8b